core/templates/pages/subtopic-viewer-page/subtopic-viewer-auth.guard.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Auth guard for Subtopic viewer page.
+ */
+
+import {Injectable} from '@angular/core';
+import {CanActivate} from '@angular/router';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SubtopicViewerAuthGuard implements CanActivate {
+  async canActivate(): Promise<boolean> {
+    // Return true as all users can access the Subtopic Viewer Page.
+    return true;
```

Comment:
Don't we need to call `validateAccessToSubtopicViewerPage` to verify whether the user can access the page?

Also, where is this `validateAccessToSubtopicViewerPage` used if not here?

Reply:
@Lawful2002 Looks like the authGuard for the new page is not added correctly, can you please fix this quickly? (Maybe create an issue before you start working on it!)